prow/getting_started.md

```diff
@@ -190,6 +190,21 @@ update-config: get-cluster-credentials
 Presubmits and postsubmits are triggered by the `trigger` plugin. Be sure to
 enable that plugin by adding it to the list you created in the last section.
 
+By default, services are not granted permission to run these jobs. Run the
+following to grant permission to all services to create and run Prow Jobs (NOTE: This
+command grants a permissive policy to _all_ services in the cluster. We do this
+because the only services running in the prow starter are ones that we have
+written and trust. This is not the recommended policy for most clusters.
+See [RBAC documentation](https://kubernetes.io/docs/admin/authorization/rbac/) for more details):
```

Comment:
This seems fine for now, but eventually we should switch to proper RBAC rules like the ones used by OpenShift instead of granting all components all permissions: https://github.com/kubernetes/test-infra/issues/7950#issuecomment-387048959

Reply:
Agreed, I'l try to get a pull request in that incorporates the more specific RBAC rules from that link.

Reply:
@kargakis @stevekuznetsov The RBAC rules referenced in #7950 are contained in [openshift release](https://github.com/openshift/release) which does not have a license. Could one be attached so we can integrate the RBAC rules into this version of prow?

Reply:
Opened https://github.com/openshift/release/pull/877

Reply:
> Opened openshift/release#877

Merged

Reply:
Great! Thanks! ping @paulangton